apps/webapp/src/state/send.ts

```diff
@@ -148,16 +149,19 @@ export const validateAmount = (asset: AssetBalance, amount: string): boolean =>
 export interface SendValidationFields {
   recipientErr: boolean;
   amountErr: boolean;
+  memoErr: boolean;
 }
 
 export const sendValidationErrors = (
   asset: AssetBalance | undefined,
   amount: string,
   recipient: string,
+  memo?: string,
 ): SendValidationFields => {
   return {
     recipientErr: Boolean(recipient) && !isPenumbraAddr(recipient),
     amountErr: !asset ? false : validateAmount(asset, amount),
+    memoErr: new TextEncoder().encode(memo).length > 512 - BECH32_ADDRESS_LENGTH,
```

Comment:
is this length calc correct

Reply:
According to Henry's issue it's `address+memo <= 521 bytes`. So this looks right. Can we add a code comment right above?

```diff
+ // The memo plus the address cannot exceed 512 bytes
memoErr: new TextEncoder().encode(memo).length > 512 - BECH32_ADDRESS_LENGTH,
```

Reply:
turns out this was incorrect, at least according to how wasm throws. it is using an 80-byte address in the memo and will throw between a 432-char -> 433-char memo length